.github/workflows/windows.yml

```diff
@@ -346,10 +346,14 @@ jobs:
         name: Starting test daemon
         run: |
           Write-Host "Creating service"
-          If ("${{ matrix.runtime }}" -eq "containerd") {
+          If ("${{ matrix.runtime }}" -eq "containerd" -and "${{ matrix.os }}" -eq "windows-2019") {
             $runtimeArg="--containerd=\\.\pipe\containerd-containerd"
```

Comment:
Now that the `remote.Debug.Address` no longer defaults to the system containerd, should we also pass the system container's debug socket on this CLI? I haven't looked to see how that socket is used, this is just flagging from the change in _libcontainerd/supervisor/remote_daemon_windows.go_.

Reply:
I see the Unix-side code doesn't do this either, perhaps this debug socket is only used for a locally-started containerd?

Reply:
Found this comment from containerd code `DefaultDebugAddress is the default winpipe address for pprof data` so I understand that it purely for debugging but I have no idea how to use it.

Reply:
I don't have the source in front of me, but GitHub search is telling me that Docker doesn't ever _use_ the socket. Either I'm just not searching well enough, or some refactoring has disappeared the use-cases for this and no one noticed? Or it's actually supposed to be used by someone running `ctr pprof` or another tool entirely, not by Docker itself.

Anyway, still browsing GitHub, Docker's libcontainerd/supervisor always passes in a Debug socket (pointing to a private socket in the changes you've made here parallel to the private containerd socket), and when we start containerd in the system test, without passing one in, it doesn't start listening (i.e. default is "" on Windows, Linux and Solaris).

So what we have now is fine, since we don't try and collect the pprof data in the test run, and we shouldn't see any conflicts between system-containerd and docker-private containerd instances over the socket name.

Reply:
There might be a slight separate bug related to this, as [`monitorDaemon` deletes the GRPC socket, but not the Debug socket](https://github.com/moby/moby/blob/master/libcontainerd/supervisor/remote_daemon.go#L278) when starting/restarting containerd. Probably not an issue, but if we start seeing bundled-containerd startups/restarts failing because the pprof socket already exists, then that'll be why. (Not sure if npipe semantics on Windows are different from unix sockets in this regard, my recollection is that neither _requires_ explicit removal after the ends are closed, so removing the GRPC socket in that code might just be bonus safety).

Reply:
And then there's #43901.